src/python/turicreate/config/__init__.py

```diff
@@ -329,6 +329,14 @@ def set_runtime_config(name, value):
     from .._connect import main as _glconnect
 
     unity = _glconnect.get_unity()
+
+    if name == "TURI_FILEIO_ALTERNATIVE_SSL_CERT_FILE":
+        # Expands relative path to absolute path
+        value = _os.path.abspath(_os.path.expanduser(value))
+        # Raises exception if path for SSL cert does not exist
+        if not _os.path.exists(value):
+            raise FileNotFoundError("{} does not exist.".format(value))
```

Comment:
Sadly, it looks like `FileNotFoundError` also isn't in Python 2.7. I'm open to other ideas, but I would probably just change `FileNotFoundError ` to `ValueError`.

Reply:
Ah! My bad. Fixed it. 
For the future, to test for Python2.7, how would you recommend doing it? 
I am thinking of using a virtualenv with Python2.7 and using that to make and test.

Reply:
In general testing multiple Python versions using virtualenv is the way to go. However it's difficult to test local TuriCreate changes across multiple Python versions; when setting up your local environment it gets bound to a particular Python version (this happens when you run `./configure`).

I think what happened here is a bit of an anomaly. Usually getting code to run in both Python 2 and 3 is pretty easy. Usually if code runs in Python 3, it will run in Python 2.